integration/spark/src/main/scala/org/apache/carbondata/spark/util/GlobalDictionaryUtil.scala

```diff
@@ -705,7 +709,7 @@ object GlobalDictionaryUtil extends Logging {
           // check result status
           checkStatus(carbonLoadModel, sqlContext, model, statusList)
         } else {
-          logInfo("have no column need to generate global dictionary in Fact file")
+          logInfo("have no column need to generate global dictionary in source data files")
```

Comment:
@QiangCai ...Please modify the logger message as below:
"No column found for generating global dictionary in source data files"